prizy/urls.py

```diff
@@ -30,7 +30,7 @@
     path('verify/', verify_jwt_token, name='jwt_verify'),
     # API routes
     path('api/', include('accounts.urls')),
-    path('api/', include('events.urls'))
+    path('api/events/', include('events.api.urls'), name='events-api'),
```

Comment:
Do we really need this change?

Reply:
OK, I checked your repo and it makes sense now, but why overcomplicate things? We could move everything to the events root directory!

Reply:
If this is making confusion I am going to fix that soon, is a convention I follow when it comes for APIs

Reply:
No, it's a great convention actually, but we don't have that many endpoints to handle, so it might be a bit "over-engineered" in this case. And since we didn't have that structure in `accounts`, I wanted to be consistent throughout the codebase.

We can definitely reorganize the api structure, but after discussion, to make sure we're all in the same page!

Check [PB-3 on Trello](https://trello.com/c/nLdkCnjt/2-pb-3-add-endpoint-to-list-meetup-events-to-registered-users), I've added some comments there